deploy/charts/harvester/templates/cattle-system.yaml

```diff
@@ -15,10 +15,39 @@ spec:
   secrets:
     - name: harvester-additional-ca
       path: /ca
-  version: v1.0.2
+  version: v1.1.0
   upgrade:
     image: {{ .Values.generalJob.image.repository }}:{{ .Values.generalJob.image.tag }}
     command: ["/bin/sh", "-c"]
     args:
-      - cp /ca/additional-ca.pem /host/etc/pki/trust/anchors;
-      - chroot /host update-ca-certificates
+      - echo "Update additional ca ...";
+        cp /ca/additional-ca.pem /host/etc/pki/trust/anchors;
+        chroot /host update-ca-certificates;
+        echo "Done";
+---
+apiVersion: upgrade.cattle.io/v1
+kind: Plan
+metadata:
+  name: sync-containerd-registry
+  namespace: cattle-system
+spec:
+  concurrency: 1
+  nodeSelector:
+    matchLabels:
+      harvesterhci.io/managed: "true"
+  serviceAccountName: system-upgrade-controller
+  secrets:
+    - name: harvester-containerd-registry
+      path: /registries
+  version: v1.1.0
+  upgrade:
+    image: {{ .Values.generalJob.image.repository }}:{{ .Values.generalJob.image.tag }}
+    command: ["/bin/sh", "-c"]
+    # We can't use systemctl in docker image.
+    # ref: https://stackoverflow.com/questions/49285658/how-to-solve-docker-issue-failed-to-connect-to-bus-no-such-file-or-directory
+    args:
+      - echo "Update /etc/rancher/rke2 ...";
```

Comment:
When we first apply the plan, the plan will run once even the registry setting is empty.
Could you see if there is a way to avoid the first restart?

Reply:
Updated it! Thank you.

Reply:
Thanks!